.github/CODEOWNERS

```diff
@@ -644,3 +644,5 @@
 /internal/third_party/client-go         @DataDog/container-platform
 /internal/third_party/kubernetes        @DataDog/container-integrations
 /internal/third_party/golang/           @DataDog/container-integrations
+
+/Dockerfiles/agent/bouncycastle-fips    @DataDog/agent-metrics-logs
```

Comment:
The `CODEOWNERS` file might not be fully strictly ordered but I think it makes sense to keep the current structure and move this new line close to the other lines about `Dockerfiles` around line 252: https://github.com/DataDog/datadog-agent/blob/356577911f747aecf25691568e570afb96ba2625/.github/CODEOWNERS#L247-L252